src/Components/Homepage.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import styled from 'styled-components';
+import Section from './home/Section';
+import Calendar from './home/Calendar';
+
+export default function Homepage() {
```

Comment:
You could add tests for each component too.  Test that the component mounts and doesn't blow up, renders right stuff, etc